### ISSUE TITLE
Improve how Jax array is handled

### ISSUE THREAD

**Issue by sunnyayyl** [JetBrains/lets-plot#1136]
```py
import jax.numpy as jnp
import lets_plot as lp
import numpy

(
    lp.ggplot(
        {"x": list(jnp.arange(0, 10)), "y": numpy.arange(0, 10)},
        lp.aes(x="x", y="y"),
    )


    + lp.geom_line()
).show()
```
![test](https://github.com/user-attachments/assets/2262d6fc-0ce7-44e0-bd31-b52a159c9dc2)
y axis is the expected behavior with numpy
x axis is the behavior when used with jax
(jax: https://jax.readthedocs.io/en/latest/index.html)

**Comment by alshan** [comment]
Hi, in the example above, you are passing a list of arrays. 
When the elements in the list are anything but numeric values, Lets-Plot treats them as discrete (a.k.a. categorical) values and creates a discrete axis where the labels are simple string representations of the elements in the list.

A bare jax array, however, will most likely not be recognized as a collection of numerics either. Did you try this?

**Comment by sunnyayyl** [comment]
I know I can use `numpy.asarray`, but it would be nice if the conversion can be done implicitly, so I guess this is more of a feature request.

**Comment by alshan** [comment]
Sure, thanks for the feature request.

**Comment by sunnyayyl** [comment]
I'll edit the title to better reflect that this issue is a feature request. Thanks for your help!

**Comment by sunnyayyl** [comment]
I had a look at the source code, would this issue be solved by adding to this part? 
https://github.com/JetBrains/lets-plot/blob/master/python-package%2Flets_plot%2F_type_utils.py

**Comment by sunnyayyl** [comment]
Something on the line of
```py
try:
   import jax.numpy as jnp
except:
   jnp=None

...

if (jnp and isinstance(v, jnp.ndarray))
        return _standardize_value(v.tolist())
```

**Comment by alshan** [comment]
This looks promising, thanx!

**Comment by sunnyayyl** [comment]
I'm currently working on this, I will try and open a pull request when I am done

**Comment by sunnyayyl** [comment]
![plot](https://github.com/user-attachments/assets/7fdbbfbd-6eb2-4f02-a33b-d1d97e3a101e)
